src/test/kotlin/day18/dataSource/PhotoDataSourceImplTest.kt

```
@@ -0,0 +1,10 @@
+package day18.dataSource
+
+import org.junit.Test
+
+import org.junit.Assert.*
+ class PhotoDataSourceImplTest {
+
+@Test
+ fun getPhoto() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)